PMMain/Midas/MidasGUI/keyboardData.txt

```diff
@@ -0,0 +1,4 @@
+aA bB cC dD eE fF gG hH iI jJ kK lL mM
+nN oO pP qQ rR sS tT uU vV wW xX yY zZ
+`~ 1! 2@ 3# 4$ 5% 6^ 7& 8* 9( 0) -_ =+
+[{ ]} ;: '" \| ,< .> |\ /? th is te nd
```

Comment:
what is the purpose of "th is te nd" ? Shouldnt be needed since theres no requirement for sizes of 13.